projects/curl.se/ca-certs/package.yml

```diff
@@ -0,0 +1,23 @@
+distributable: ~
+
+versions:
+  # maybe base it off the URL?
+  - 1.0.0
+
+runtime:
+  env:
+    SSL_CERT_FILE: ${{prefix}}/ssl/cert.pem
+
+build:
+  dependencies:
+    linux:
```

Comment:
we require `curl` on all platforms, it's just a coincidence that macOS provides it.

longer term we will probs “detect” system provided stuff for these sorts of scenarios.